### DISCUSSION TITLE
Save/export the parameters and Mods installed on my computer Win 11 to use on another computer. Possible ?

### DISCUSSION THREAD

**Discussion by sypqys** [ramensoftware/windhawk-mods#437] [Q&A] [answered]
Hi !

I don't find in paramaters this possibility...
If exist ?

Thanks

**Comment by m417z** [comment, marked as answer]
Hi, 

I was thinking about adding an export/import feature, but currently, there's no official way. For the non-portable version, both files from the program data folder and registry entries will have to be transferred.
I haven't tried it, but I believe that it's enough to copy over the following:
```
%ProgramData%\Windhawk\Engine\Mods
%ProgramData%\Windhawk\ModsSource
HKLM\SOFTWARE\Windhawk\Engine\Mods
HKLM\SOFTWARE\Windhawk\Engine\ModsWritable
```

Let me know whether that works for you.
